packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -13,11 +16,7 @@ export enum NotificationPromptSource {
 
 const DISMISS_BROWSER_PERMISSION = 'dismissBrowserPermission';
 
-type DismissBrowserPermissions = {
-  [NotificationPromptSource.NotificationList]: boolean;
-  [NotificationPromptSource.NewComment]: boolean;
-  [NotificationPromptSource.NewSource]: boolean;
-};
+type DismissBrowserPermissions = Record<NotificationPromptSource, boolean>;
```

Comment:
@el just made it use a record so new introduction of `NotificationPromptSource` will automatically result to error if the dev won't add the new property in the implementation.

Reply:
Why did you abstract it into a type and not directly in the const?
We don't really have a rule for it, but seeing as it's one singular type it might make sense to keep it inline?

Let me know what you think.

Reply:
It was already a type, I just updated it to be a record to make it type-strict.